crates/ra_hir/src/code_model_impl/module.rs

```diff
@@ -56,6 +57,16 @@ impl Module {
         source_map.get(&source, import)
     }
 
+    pub(crate) fn impl_source_impl(
+        &self,
+        db: &impl HirDatabase,
+        impl_id: ImplId,
+    ) -> TreeArc<ast::ImplBlock> {
+        let source_map = db.impls_in_module_source_map(self.clone());
```

Comment:
this could be `*self`. `Module`is `Copy` now, and we probably should just change all those `&self` to `self.`